test/app_test.go

```diff
@@ -0,0 +1,55 @@
+package test
+
+import (
+	"openpitrix.io/openpitrix/test/client/app_service"
+	"openpitrix.io/openpitrix/test/models"
+	"log"
+	"testing"
+	"github.com/go-openapi/runtime"
```

Comment:
The order should be standard, other, our package, so here should be sorted. 
And then go fmt, indent is 1 tab=8 space

Reply:
i saw the editorconfig is:
```plain
[*.{go,proto}]
charset = utf-8
indent_style = tab
tab_width = 4
```

Reply:
Sorry, I was misremembered.